### ISSUE TITLE
[a1b64e]The rule "Focusable element has no keyboard trap via standard navigation" Failed Examples 2 & 3 buttons are unreachable, rather than trapped

### ISSUE THREAD

**Issue by shunguoy** [auto-wcag/auto-wcag#2375]
Please describe the issue with references and relevant examples where necessary.

In the Failed Example 2 & 3, the button is not reachable through Tab, however, the focus can always back to the address bar through Shift + Tab, so it's not a trap.
